fili-navi/src/test/resources/testApplicationConfig.properties

```diff
@@ -1,7 +1,7 @@
-# Copyright 2017 Yahoo Inc.
+# Copyright 2016 Yahoo Inc.
 # Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
 
-#The name of this module (required)
-moduleName = fili-navi
```

Comment:
Module name belongs in moduleConfig.properties not testApplicationConfig.properties.  This was a bug.